### PR TITLE
Re-order the upsell

### PR DESCRIPTION
**Pull request by jfsoul**
## What does this change?
Adds ordering to the upsell cards of the new thank-you page.

## Screenshots
image

## What is the value of this and can you measure success?
Card ordering matches requirement once more.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCardList.js [hunk 1]
```diff
@@ -22,19 +22,35 @@ const getConsents = (): Promise<ConsentWithState[]> => {
...
+        'green-light',
+        'lab-notes',
+    ];
+
     return Promise.all([userConsents, newsLetterConsents]).then(
-        ([fetchedUserConsents, fetchedNewsLetterConsents]) => [
-            ...fetchedUserConsents,
-            ...fetchedNewsLetterConsents,
-        ]
+        ([fetchedUserConsents, fetchedNewsLetterConsents]) =>
```
Comment (walaura) [inline static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCardList.js]: might be worth going back to the prior interface with the list of promises to avoid all this rejiggling?

Comment (jfsoul) [reply static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCardList.js]: Yes I did wonder about that but I was lazy and didn't work our properly whether that would cause lots of requests to fetch each consent individually.

**Review by Mullefa — APPROVED** [review]
In terms of trying to refine the type via a filter, it seems that this [isn't currently possible](https://github.com/facebook/flow/issues/1414). Refinement of types aside, a filter would also be useful to ensure all the consents are well-defined (even if it's not encoded in the type). However, flow also complained about removing undefined elements:
```javascript
consentsPromise.then(consents => consents.filter(Boolean))
```

It's fortunate at least, that even though `States.consents` is modelled as `ConsentWithState[]`, the value of the returned promise -  `(?ConsentWithState)[]` - is allowed by flow for this field 😅. 

So basically... I approve since it seems like a reasonable way of navigating through all of flow's checks, but it might be worth seeking the input of someone with a deeper knowledge of flow on Monday. @walaura ?
